Sources/AirbnbSwiftFormatTool/AirbnbSwiftFormatTool.swift

```diff
@@ -90,13 +123,16 @@ struct AirbnbSwiftFormatTool: ParsableCommand {
       arguments += ["--swiftversion", swiftVersion]
     }
 
-    let swiftFormat = Process()
-    swiftFormat.launchPath = swiftFormatPath
-    swiftFormat.arguments = arguments
-    return swiftFormat
-  }()
+    return Command(
```

Comment:
```suggestion
    return .init(
```
Nit: I think this is preferable since it makes refactoring easier. Could be a style guide rule one day :)

Reply:
Not sure I agree -- I personally prefer using the non `.init(` form (with the type name) in the majority of cases because I find it makes the code easier to read. (I've seen lots of complex invocations with multiple `.init`s before where the code would be a lot easier to understand if the type names were written out explicitly instead of using the bare `.init`).

Reply:
I can see "prefer .init where possible" being in alignment with our style guide principles (where we generally omit things that are unnecessary) but I find it a bit hard to swallow subjectively, and I think there are many cases where `.init` would read as less idiomatic.

Reply:
Thanks for sharing your POV on this. _If_ we adopted a "prefer .init where possible" rule it means that we need to rely more on IDE features for context, which can admittedly be finicky.